CMakeLists.txt

```diff
@@ -0,0 +1,256 @@
+    OpenHome/Net/Device/DviProviderSubscriptionLongPoll.cpp 
+    OpenHome/Net/Device/Providers/DvOpenhomeOrgSubscriptionLongPoll1.cpp 
+    OpenHome/Net/Device/DviServer.cpp 
+    OpenHome/Net/Device/Upnp/DviServerUpnp.cpp 
+    OpenHome/Net/Device/Upnp/DviServerWebSocket.cpp 
+    OpenHome/Net/Device/Lpec/DviProtocolLpec.cpp 
+    OpenHome/Net/Device/Lpec/DviServerLpec.cpp 
+    OpenHome/Net/Device/DviService.cpp 
+    OpenHome/Net/Device/Upnp/DviSsdpNotifier.cpp 
+    OpenHome/Net/Device/DviStack.cpp 
+    OpenHome/Net/Device/DviSubscription.cpp 
+    OpenHome/Net/Device/DviPropertyUpdateCollection.cpp 
+    OpenHome/Net/Device/DvProvider.cpp 
+    OpenHome/Net/Bindings/C/Device/DvProviderC.cpp 
+    OpenHome/Net/Device/DvServerUpnp.cpp 
+    OpenHome/Net/Bindings/C/Device/DvServerUpnpC.cpp 
+    OpenHome/Net/ControlPoint/Upnp/DeviceXml.cpp 
+    OpenHome/Net/Error.cpp 
+    OpenHome/Net/ControlPoint/Upnp/EventUpnp.cpp 
+    OpenHome/Exception.cpp 
+    OpenHome/Fifo.cpp 
+    OpenHome/FileAnsii.cpp 
+    OpenHome/FileBrx.cpp 
+    OpenHome/FileStream.cpp 
+    OpenHome/Net/Globals.cpp 
+    OpenHome/Http.cpp 
+    OpenHome/Net/Device/Bonjour/mDNSCore/DNSCommon.c 
+    OpenHome/Net/Device/Bonjour/mDNSCore/DNSDigest.c 
+    OpenHome/Net/Device/Bonjour/mDNSCore/mDNS.c 
+    OpenHome/Net/Device/Bonjour/mDNSCore/uDNS.c 
+    OpenHome/Net/Device/Bonjour/MdnsPlatform.cpp 
+    OpenHome/Net/Device/Bonjour/MdnsProvider.cpp 
+    OpenHome/md5.c 
+    OpenHome/NetworkAdapterList.cpp 
+    OpenHome/Network.cpp 
+    OpenHome/Parser.cpp 
+    OpenHome/Printer.cpp 
+    OpenHome/Net/ControlPoint/Upnp/ProtocolUpnp.cpp 
+    OpenHome/Queue.cpp 
+    OpenHome/Net/Service.cpp 
+    OpenHome/Net/Bindings/C/ServiceC.cpp 
+    OpenHome/sha1.c 
+    OpenHome/Net/Ssdp.cpp 
+    OpenHome/Net/SsdpDv.cpp
+    OpenHome/Env.cpp 
+    OpenHome/Stream.cpp 
+    OpenHome/Net/Subscription.cpp 
+    OpenHome/Terminal.cpp 
+    OpenHome/Thread.cpp 
+    OpenHome/Timer.cpp 
+    OpenHome/Uri.cpp 
+    OpenHome/Net/XmlParser.cpp 
+    OpenHome/Net/ControlPoint/Upnp/XmlFetcher.cpp 
+    OpenHome/Net/OhNet.cpp 
+    OpenHome/Net/OhNetCp.cpp 
+    OpenHome/Net/OhNetDv.cpp 
+    OpenHome/Net/OhNetCombined.cpp 
+    OpenHome/Net/Bindings/C/OhNetC.cpp
+    OpenHome/Net/Bindings/C/OhNetCCp.cpp 
+    OpenHome/Net/Bindings/C/OhNetCDv.cpp 
+    OpenHome/Net/Bindings/C/OhNetCCombined.cpp 
+    Os/OsWrapper.cpp 
+    Os/Posix/File.cpp 
+    Os/Posix/TerminalOs.cpp 
+    Os/Posix/Os.c 
+    Os/Posix/SignalHandlers.cpp 
+    OpenHome/Shell/Shell.cpp 
+    OpenHome/Shell/InfoProvider.cpp 
+    OpenHome/Shell/ShellCommandDebug.cpp 
+    OpenHome/Net/Bindings/Android/jni/ifaddrs-android.cpp )
+
+
+add_library( # Specifies the name of the library.
+             ohNet
+
+             # Sets the library as a shared library.
+             SHARED
+
+             # Provides a relative path to your source file(s).
+             ${SourcesMain} )
+
+
+add_custom_target( IncludeCopy )
```

Comment:
This code section is to duplicate the header file copying in standard build.  Unfortunately ninja generator in windows generated massive lines, so I added the lines in chunks which didn't cause windows to barf.  Unfortunately ugly